### PR TITLE
notification timestamps and formatting notification button

### PR DESCRIPTION
**Pull request by mekkyz**
Fixes https://github.com/ComPlat/chemotion/issues/12
Fixes https://github.com/ComPlat/chemotion/issues/20

- used "prettier" to reformat the code.
- solved eslint issues
- getting the timezone String from backend to correctly present the local time
- DidMount has now startActivityDetection to make the notification button always available
- Did some ui change on the notification button because its size and centering were off

What is new for users:
- the notification button is now a bell instead of the messaging icon and be always shown in the default status when notification = 0.
- it will change to yellow and notification counter will start counting whenever there are notifications
- the text "message" is now "notification"
- the button has now the right size comparable to the button next to it.
- the icon is now centered in the button.

This is how it was when there were no notifications.

And now:

When there are notifications

And now:

### CODE REVIEW

File: app/packs/src/utilities/timezoneHelper.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  const [hours, minutes, seconds] = timePart.split(':');
+
+  // format date to js readable date
+  const formattedDate = `${year}-${month}-${day}T${hours}:${minutes}:${seconds}`;
+
+  const creationDateObj = new Date(formattedDate);
+
+  // local timezone offset
+  const timeZoneOffset = creationDateObj.getTimezoneOffset();
+  const offsetHours = Math.floor(Math.abs(timeZoneOffset / 60));
```
Comment (ptrxyz) [inline app/packs/src/utilities/timezoneHelper.js:13]: to calculate the offset, simply use `Math.trunc` or `parseInt`. Those do the same but preserve the sign so you can get rid of the comparisons in line 19 and 23.

File: spec/javascripts/utils/timezoneHelper.spec.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  it('should correctly format the date', () => {
+    const input = '23.06.2023, 12:00:00';
+    const expectedOutput = 'Friday, June 23, 2023 at 12:00 PM';
+
+    const result = formatDate(input);
+
+    expect(result).toEqual(expectedOutput);
+  });
+
+  it('should handle different date formats', () => {
```
Comment (ptrxyz) [inline spec/javascripts/utils/timezoneHelper.spec.js:14]: Isnt this the same test as the one before just with a different date but the same format?

**Review by ptrxyz — CHANGES_REQUESTED** [review]

File: app/packs/src/utilities/timezoneHelper.js (hunk 1 above)
Comment (mekkyz) [reply app/packs/src/utilities/timezoneHelper.js:13]: corrected

File: spec/javascripts/utils/timezoneHelper.spec.js (hunk 2 above)
Comment (mekkyz) [reply spec/javascripts/utils/timezoneHelper.spec.js]: deleted the second one. I was testing with different dates

**Review by PiTrem — APPROVED** [review]

**Pull request merged**